api/registration/models.py

```diff
@@ -52,15 +65,6 @@ class User(AbstractUser, PermissionsMixin):
     date_joined = models.DateTimeField(auto_now_add=True)
 
```

Comment:
Please remove the type default parameter and these fields (is_staff, is_superuser)